examples/graham-fundamentals/README.md

```diff
@@ -0,0 +1,8 @@
+This is an example algorithm that shows how you can translate some of the principles Benjamin Graham laid out in his book, The Intelligent Investor, into a script that trades in a live environment.
+
+To connect pylivetrader to your existing Alpaca live or paper account, you can see the full instructions[here].(https: // github.com/alpacahq/pylivetrader/tree/master/examples)
```

Comment:
Isn't this funny space?

Reply:
Oh, this shouldn't even be in the pull request - weird that it is